slo.go

```diff
@@ -0,0 +1,155 @@
+package gapi
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+)
+
+var sloPath string = "/api/plugins/grafana-slo-app/resources/v1/slo"
+
+type Slos struct {
+	Slos []Slo `json:"slos"`
+}
+
+type Slo struct {
```

Comment:
These have to match the structs from the server, I assume these are copies. Should we leave a breadcrump, comment, to make clear to this reader where they were copied from (i.e. in case of updates).

Even better would be if we could refer to a swagger/cue/proto file since these are effectively API defined structs.

I'm guessing we don't have a great solution there, maybe a comment? or not